examples/3d/generate_custom_mesh.rs

```diff
@@ -122,7 +122,8 @@ fn create_cube_mesh() -> Mesh {
     Mesh::new(PrimitiveTopology::TriangleList, RenderAssetUsages::MAIN_WORLD | RenderAssetUsages::RENDER_WORLD)
     .with_inserted_attribute(
         Mesh::ATTRIBUTE_POSITION,
-        // Each array is an [x, y, z] coordinate in local space. Bevy uses a right-handed Y-up coordinate system.
+        // Each array is an [x, y, z] coordinate in local space.
+        // The camera coordinate space is right-handed x-right, y-up, z-back. This means "forward" is -Z.
```

Comment:
This example isn't about the camera, so the camera's axis-orientation doesn't seem especially pertinent to me. Is there a reason why it belongs here?

Reply:
The issue was specifically about making this comment match a similar comment in the Camera3d docs. Personally, I think it's a helpful change to explicitly spell out which axis points where for people to understand the mesh below.

Reply:
It's relevant in the sense that when making a custom mesh, you might care about it's orientation when spawning and knowing the camera coordinate is relevant for that. I agree that it's not the ideal place, but it was added by a user that ran into this problem and that's where they were looking for it.